### PR TITLE
OP-780 - OH core make spring use XXX_fields

### PR DESCRIPTION
**Pull request by mwithi**
See OP-780.

Paired with https://github.com/informatici/openhospital-gui/pull/1483.

Co-authored-by: Andrei Dodu <andrei.dodu@hotmail.it>
Co-authored-by: Alessandro Domanico <alessandro.domanico@yahoo.it>
Co-authored-by: David B Malkovsky <david@malkovsky.org>

### CODE REVIEW

File: src/main/java/org/isf/admission/service/AdmissionIoOperationRepositoryImpl.java [hunk 1]
```diff
@@ -39,23 +39,22 @@
...
-			+ " where p.PAT_ID = :param0 "
-			+ " and ( ( p.PAT_DELETED='N' ) or ( p.PAT_DELETED is null ) )";
-
+	private static String nativeQueryTerms = "SELECT * from oh_patient as p  "
+					+ " left join (select * from oh_admission where ADM_IN=1 and ( ADM_ACTIVE=1 ) ) as a on p.PAT_ID=a.ADM_PAT_ID "
+					+ " where p.PAT_ACTIVE=1 "
+					+ " and ( lower(concat_ws(' ', p.PAT_ID, p.PAT_SNAME, p.PAT_FNAME, p.PAT_NAME, p.PAT_NOTE, p.PAT_TAXCODE, p.PAT_CITY, p.PAT_ADDR, p.PAT_TELE)) like :param0 ) "
+					+ " order by p.PAT_ID desc";
+
+			private static String nativeQueryRanges = "SELECT * from oh_patient as p  "
```
Comment (dbmalkovsky) [inline src/main/java/org/isf/admission/service/AdmissionIoOperationRepositoryImpl.java:48]: Why is this String and the next String indented so far into the file and not aligned with the String field in line 42?

File: src/main/java/org/isf/patient/model/Patient.java [hunk 2]
```diff
@@ -552,13 +550,17 @@ public String getSearchString() {
 		sbName.append(' ');
 		sbName.append(getCity().toLowerCase());
 		sbName.append(' ');
-		if (getAddress() != null) sbName.append(getAddress().toLowerCase()).append(' ');
-		if (getTelephone() != null) sbName.append(getTelephone()).append(' ');
-		if (getNote() != null) sbName.append(getNote().toLowerCase()).append(' ');
-		if (getTaxCode() != null) sbName.append(getTaxCode().toLowerCase()).append(' ');
+		if (getAddress() != null)
```
Comment (dbmalkovsky) [inline src/main/java/org/isf/patient/model/Patient.java:553]: If this is going to be reformatted then please add the project standard of using opening and closing braces for `if` statements; thanks

File: src/main/java/org/isf/utils/db/Auditable.java [hunk 3]
```diff
@@ -72,7 +73,7 @@ public LocalDateTime getCreatedDate() {
         return TimeTools.truncateToSeconds(createdDate);
     }
 
-    public int getActive() {
+    public Integer getActive() {
```
Comment (dbmalkovsky) [inline src/main/java/org/isf/utils/db/Auditable.java:76]: Personally I think that raw types (`int`) should be used in place of Objects as it just causes unnecessary boxing and unboxing of values......

File: src/main/java/org/isf/admission/service/AdmissionIoOperationRepositoryImpl.java (hunk 1 above)
Comment (mwithi) [reply src/main/java/org/isf/admission/service/AdmissionIoOperationRepositoryImpl.java]: Fixed, thanks!

File: src/main/java/org/isf/patient/model/Patient.java [hunk 4]
```diff
@@ -262,12 +268,12 @@ public Patient(String firstName, String secondName, LocalDate birthDate, int age
...
-	public Patient(int code, String firstName, String secondName, String name, LocalDate birthDate, int age, String agetype, char sex,
-			String address, String city, String nextKin, String telephone, String note,
-			String motherName, char mother, String fatherName, char father,
-			String bloodType, char economicStatut, char parentTogether, String taxCode,
-			String maritalStatus, String profession) { //Changed EduLev with bloodType
+
+	public Patient(int code, String firstName, String secondName, String name, LocalDate birthDate, int age,
+			String agetype, char sex, String address, String city, String nextKin, String telephone, String note,
+			String motherName, char mother, String fatherName, char father, String bloodType, char economicStatut,
+			char parentTogether, String taxCode, String maritalStatus, String profession) { // Changed EduLev with
```
[the remark this answers is not in the document]
Comment (mwithi) [reply src/main/java/org/isf/patient/model/Patient.java]: Fixed, thanks!

File: src/main/java/org/isf/utils/db/Auditable.java (hunk 3 above)
Comment (mwithi) [reply src/main/java/org/isf/utils/db/Auditable.java:76]: Agreed, thanks!

**Comment by mwithi** [comment]
Close because OP-780 has been canceled.

**Pull request closed without merging**